pkg/audit/manager.go

```diff
@@ -48,22 +48,24 @@ type AuditManager struct {
 }
 
 type auditResult struct {
-	cname       string
-	cnamespace  string
-	cgvk        schema.GroupVersionKind
-	capiversion string
-	rkind       string
-	rname       string
-	rnamespace  string
-	message     string
+	cname             string
+	cnamespace        string
+	cgvk              schema.GroupVersionKind
+	capiversion       string
+	rkind             string
+	rname             string
+	rnamespace        string
+	message           string
+	enforcementaction string
```

Comment:
It looks like this change is adding enforcementaction. Maybe we need to re-sync the master branch into the mutation branch?